submissions/2019-07-25-phenyl-set/compute.json

```diff
@@ -0,0 +1,87 @@
+{
+  "qc_specifications": {
+    "openff-1.0.0": {
+      "method": "openff-1.0.0",
+      "basis": "smirnoff",
+      "program": "openmm",
+      "spec_name": "openff-1.0.0",
+      "spec_description": "default openff-1.0.0 spec",
+      "store_wavefunction": "none"
+    },
+    "openff-1.2.0": {
+      "method": "openff-1.2.0",
+      "basis": "smirnoff",
+      "program": "openmm",
+      "spec_name": "openff-1.2.0",
+      "spec_description": "default openff-1.2.0 spec",
+      "store_wavefunction": "none"
+    },
+    "openff-1.2.1": {
+      "method": "openff-1.2.1",
+      "basis": "smirnoff",
+      "program": "openmm",
+      "spec_name": "openff-1.2.1",
+      "spec_description": "default openff-1.2.1 spec",
+      "store_wavefunction": "none"
+    },
+    "gaff-2.11": {
+      "method": "gaff-2.11",
+      "basis": "antechamber",
+      "program": "openmm",
+      "spec_name": "gaff-2.11",
+      "spec_description": "default gaff-2.11 spec",
+      "store_wavefunction": "none"
+    }
+  },
+  "dataset_name": "OpenFF Substituted Phenyl Set 1",
+  "dataset_tagline": "OpenForcefield TorsionDrives.",
+  "dataset_type": "TorsiondriveDataset",
+  "maxiter": 200,
+  "driver": "gradient",
+  "scf_properties": [
+    "dipole",
+    "quadrupole",
+    "wiberg_lowdin_indices",
+    "mayer_indices"
+  ],
+  "priority": "normal",
+  "description": "A TorsionDrive dataset using geometric.",
+  "dataset_tags": [
+    "openff"
+  ],
+  "compute_tag": "openff",
+  "metadata": {
```

Comment:
Would like to see the metadata look a little more complete/representative. Was this generated from scratch, or pulled from the existing dataset?

Reply:
I pulled this JSON from the `compute.json` of the Benchmark Ligands set. I modified it where needed. What would you like here?

Reply:
Added some more specific metadata. Thanks for pointing this out @trevorgokey!